EU4toV2/Source/V2World/V2World.cpp

```diff
@@ -80,6 +81,212 @@ V2World::V2World(const EU4::world& sourceWorld, const mappers::IdeaEffectMapper&
 	output(potentialGPs);
 }
 
+class BucketList : private commonItems::parser

```

Comment:
Bucketlist should go in its own h/cpp files that V2World can include.

Reply:
Done; it was only meant to be a tiny struct with like two vectors...